src/components/Notifications/TimeConfig.tsx

```diff
@@ -15,8 +16,17 @@ export const TimeConfigComponent: React.FunctionComponent = () => {
     const [ radioSelect, setRadioSelect ] = React.useState(false);
     const [ showCustomSelect, setShowCustomSelect ] = React.useState(false);
 
+    const getTimePreference = useGetTimePreference();
     const saveTimePreference = useUpdateTimePreference();
 
+    const timePref = useMemo(() => {
+        if (getTimePreference.payload?.status === 200) {
+            return getTimePreference.payload.value;
+        }
+
+        return undefined;
```

Comment:
Could we add some sort of loading indicator when we get undefined? Maybe replace the components with a Skeleton https://patternfly-react.surge.sh/components/skeleton

Reply:
I will be adding toast notifications to the page on success/failure with this [ticket](https://issues.redhat.com/browse/RHCLOUD-25364). With that, do you also want a skeleton?